<ui version="4.0" >
 <class>WidgetAccounts</class>
 <widget class="QWidget" name="WidgetAccounts" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Accounts</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QListWidget" name="fList" />
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QToolButton" name="fBtnNew" >
         <property name="text" >
          <string>New...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Type:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="fType" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <item>
          <property name="text" >
           <string>Account</string>
          </property>
          <property name="icon" >
           <iconset resource="application.qrc" >:/icons/icon_account.png</iconset>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Group</string>
          </property>
          <property name="icon" >
           <iconset resource="application.qrc" >:/icons/icon_accountgroup.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="fName" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Password:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="fPassword" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Group:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QComboBox" name="fGroup" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="fGroupBasic" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Basic</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="horizontalSpacing" >
             <number>0</number>
            </property>
            <property name="verticalSpacing" >
             <number>0</number>
            </property>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="fBasicGetUserInfo" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Get User Info</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QCheckBox" name="fBasicPostNews" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Post News</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="fBasicBroadcast" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Broadcast</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QCheckBox" name="fBasicClearNews" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Clear News</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="fBasicSetTopic" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Set Topic</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="fGroupFiles" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Files &amp;&amp; Folders</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="horizontalSpacing" >
             <number>0</number>
            </property>
            <property name="verticalSpacing" >
             <number>0</number>
            </property>
            <item rowspan="2" row="0" column="0" >
             <widget class="QCheckBox" name="fFilesDownload" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Download</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QCheckBox" name="fFilesCreateFolders" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Create Folders</string>
              </property>
             </widget>
            </item>
            <item rowspan="3" row="1" column="1" >
             <widget class="QCheckBox" name="fFilesMoveChange" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Move &amp;&amp; Change</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="fFilesUpload" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Upload</string>
              </property>
             </widget>
            </item>
            <item rowspan="2" row="3" column="0" >
             <widget class="QCheckBox" name="fFilesUploadAnywhere" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Upload Anywhere</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="QCheckBox" name="fFilesDelete" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QCheckBox" name="fFilesViewDropBoxes" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>View Drop Boxes</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="fGroupAdmin" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Users &amp;&amp; Groups</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="horizontalSpacing" >
             <number>0</number>
            </property>
            <property name="verticalSpacing" >
             <number>0</number>
            </property>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="fUsersCreate" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Create Accounts</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QCheckBox" name="fUsersKick" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Kick Users</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="fUsersEdit" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Edit Accounts</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QCheckBox" name="fUsersBan" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Ban Users</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="fUsersDelete" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Delete Accounts</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QCheckBox" name="fUsersNoKick" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Cannot Be Kicked</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QCheckBox" name="fUsersElevate" >
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>Elevate Privileges</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="fGroupLimits" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Limits</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Download:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="fLimitDown" >
            <property name="minimumSize" >
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_7" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>KB/s</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Upoad:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="fLimitUp" >
            <property name="minimumSize" >
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="label_8" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>KB/s</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="fBtnDelete" >
         <property name="text" >
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fBtnApply" >
         <property name="text" >
          <string>Apply Changes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections/>
</ui>
